src/views/results/index.ts

```diff
@@ -50,6 +50,7 @@ class ResultSetPanelProvider {
           this._view.webview.postMessage({
             command: `rows`,
             rows: queryResults.data,
+            columnList: queryResults.metadata.columns.map(x=>x.name),
```

Comment:
This is likely an issue because `metadata` is not (and should not) be returned on every fetch.

Reply:
edit: looks like you solved this exact issue in PR #93

Reply:
Plucked to https://github.com/halcyon-tech/vscode-db2i/pull/90/commits/c5178ef0a0b719f6dbc7280a2386adadeb1ce21f